src/sass/disability-benefits.scss

```diff
@@ -0,0 +1,339 @@
+@import 'style';
```

Comment:
There's a lot of styling of generic html elements nested underneath other classes in here. I don't think we have any guidelines about that, but I think other apps we seem to lean more toward making classes and styling those and not relying as much on nested selectors.

I'd rather get this merged than do that kind of refactor right now, though.